[DOCS] More doc updates to remove the CLI (#7874)
great-expectations/great_expectations: docs/docusaurus/docs/guides/setup/configuring_metadata_stores/how_to_configure_a_validation_result_store_in_azure_blob_storage.md

}
```

### 5. Reference the new configuration

> To make Great Expectations look for Validation Results on the Azure store, you must set the ``validations_store_name`` variable to the name of your Azure Validations Store, which in our example is `validations_AZ_store`.

Reviewer: ```suggestion
To make Great Expectations look for Validation Results in the Azure Store, you must set the ``validations_store_name`` variable to the name of your Azure Validations Store. In this example, it is `validations_AZ_store`.
```
